home.cpp

```diff
@@ -5,5 +5,5 @@ using namespace std;
 int main() {
     cout << "Hello World" << endl;
     cout << "Welcome to Git" << endl;
-    cout << "Welcome to Git" << endl;
```

Comment:
Thanks for catching that typo; I meant to do it, but using GitHub is amazing. It actually better with peeps I know lol. Anyways good code!